packages/artillery-plugin-publish-metrics/lib/open-telemetry/index.js

```diff
@@ -459,23 +482,22 @@ class OTelReporter {
     }
 
     try {
-      const url = new URL(req.url);
       span.setAttributes({
-        'url.full': url.href,
-        'url.path': url.pathname,
-        'server.address': url.hostname,
-        // We set the port if it is specified, if not we set to a default port based on the protocol
-        'server.port': url.port || (url.protocol === 'http' ? 80 : 443),
-        'http.request.method': req.method,
-        'http.response.status_code': res.statusCode
+        [SemanticAttributes.HTTP_STATUS_CODE]: res.statusCode,
+        [SemanticAttributes.HTTP_REQUEST_CONTENT_LENGTH]:
+          res.request.options.headers['content-length'],
+        [SemanticAttributes.HTTP_FLAVOR]: res.httpVersion,
+        [SemanticAttributes.HTTP_USER_AGENT]:
+          res.request.options.headers['user-agent']
       });
 
       if (res.statusCode >= 400) {
-        span.setStatus({ code: SpanStatusCode.ERROR });
-      }
-      if (this.traceConfig?.attributes) {
-        span.setAttributes(this.traceConfig.attributes);
+        span.setStatus({
+          code: SpanStatusCode.ERROR,
+          message: res.statusMessage
```

Comment:
is there a reason to remove this? I think we should still set the message if available, no?